### ISSUE TITLE
Add an alumni section to the contribute page

### ISSUE THREAD

**Issue by jennwrites** [hoodiehq/hood.ie#238]
Part of https://github.com/hoodiehq/hood.ie/issues/234
### :octocat: Starter Issue

We would like to take the opportunity to invite someone who did not yet contribute to Hoodie to work on this issue. Follow [@YourFirstPR](https://twitter.com/yourfirstPR) on Twitter for more of these issues across many open source projects :relaxed:

---
### :computer: The Task

In preparation for the Camp Release we are trying to update the hood.ie site. This specific task involves updating the Hoodie community page to reflect our current contributors list. :dog:
- [x] Claimed (@May-8)
- [ ] Go to the [Community Page](https://github.com/hoodiehq/hood.ie/blob/gh-pages/community/index.html)
- [ ] Where the code is displayed (below a grey field) click on the pencil icon to the very right of the screen. 
  <img width="229" alt="screen shot 2016-02-23 at 17 50 39" src="https://cloud.githubusercontent.com/assets/2582805/13269553/641360f6-da56-11e5-8a5e-79dacd9ef39c.png">
- [ ] Make your changes in the text field.
  - [ ] Create an alumni section between `COMMUNITY` and `CONTRIBUTORS`
  - [ ] Move the following folks to this section: Lena, Jan-Erik, Caolan, Katrin, Sven, Julia Schmidt
- [ ] Describe your edits briefly under where it says **Commit changes**.
  ![Commit changes](https://cldup.com/-uX5DjuDMN-1200x1200.png)
- [ ] Select the option to create a patch or a branch to begin the Pull Request process. 
- [ ] You are done :clap: comment below that your pull request is ready for review :tada:

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)

**Comment by gr2m** [comment]
:bulb: idea:
maybe we could add paragraph to each alumni describing what they contributed to Hoodie?

**Comment by varjmes** [comment]
> maybe we could add paragraph to each alumni describing what they contributed to Hoodie?

Maybe we do this issue first, and then create a separate issue for CONTENT in the editorial repo? Because I think this can be done by someone who doesn't necessarily like writing :)

**Comment by gr2m** [comment]
Agree :)

**Comment by sxosxo** [comment]
Hi! I'm happy to take this one.

**Comment by finnp** [comment]
@SonjaHeinen Great :) I've updated the post to show that you are working on this. Let us know if you have any questions!

**Comment by sxosxo** [comment]
@finnp Thanks. Well, I'm pretty sure the code I just wrote is fine. However, I'm running into quite strange Github magic that makes the changes look like a brilliant mess. Not sure if I should submit anyway..?
Hope these screenshots explain my issue:

![githubmagic](https://cloud.githubusercontent.com/assets/11015014/14138751/34515cd0-f66f-11e5-92a1-142ebd3ad769.jpg)

**Comment by finnp** [comment]
Hey @SonjaHeinen, this looked like a Wat to me too, but it's not a problem. It's just that GitHub detected the changes differently than you actually did. The end result is the same. If you delete lines 28 to 45, it is the same result as deleting 31 to 48. This is because the lines 28 to 30 are the same as 46 to 48. Do you understand what I mean? I'm typing this in mobile, so sorry if my explanation is a bit off.

So yes you can submit it anyway :)

**Comment by sxosxo** [comment]
@finnp Yeah. I figured that's what happened. In that case, I'm curious how the code review turns out ;-)

**Comment by gr2m** [comment]
@SonjaHeinen hey Sonja, did you submit a pull request? Do you need any help to finish things up?

**Comment by sxosxo** [comment]
@gr2m Thought I made a pull request weeks ago o_0
Currently, not sure where the code went...

**Comment by gr2m** [comment]
@SonjaHeinen it’s here: https://github.com/SonjaHeinen/hood.ie/tree/patch-1

I think you missed to click the "Start Pull Request" button at the end, it’s a bit confusing, I miss that too at times

**Comment by May-8** [comment]
I would like to work on this.

**Comment by gr2m** [comment]
okay all yours :)
